crates/metadata/src/tests.rs

```diff
@@ -63,13 +63,11 @@ fn spec_contract_json() {
                         vec!["i32"].into_iter().map(AsRef::as_ref),
                     ))
                     .done()])
-                .docs(Vec::new())
```

Comment:
Because it requires a type hint now, this might technically still be an API change introducing regressions somewhere...